### ISSUE TITLE
Crash from MPD module after coming back from idle state

### ISSUE THREAD

**Issue by tardypad** [Alexays/Waybar#713]
OS: Arch Linux
waybar: 0.9.2

mpd module config:
```
"mpd": {                    
  "format": "{stateIcon}",  
  "format-stopped": "",     
  "format-disconnected": "",
  "state-icons": {          
    "paused": "",          
    "playing": ""          
  },                        
  "tooltip": false          
},                          
```

After coming from idle state, I have probably hundred of thousands of lines (maybe depends how long the idle state lasted?) in my sway log from the mpd module with an empty text like the following
```
[2020-05-20 16:59:03.534] [warning] mpd:
```
After all those lines, there is then
```
[2020-05-20 16:59:03.534] [warning] mpd: Connection to MPD closed
[2020-05-20 16:59:03.535] [info] mpd: Connected to MPD
waybar: ../src/buffer.h:152: mpd_buffer_consume: Assertion `nbytes <= mpd_buffer_size(buffer)' failed.
```
And then waybar crashes

Let me know what info you may need more if you cannot reproduce

**Comment by tardypad** [comment]
I just open my laptop back from a suspend overnight and this time I got 2GB of logs
```
[2020-06-01 10:50:28.735] [warning] mpd: home
```

**Comment by Alexays** [comment]
You only get that log?
```
[2020-05-20 16:59:03.534] [warning] mpd: Connection to MPD closed
[2020-05-20 16:59:03.535] [info] mpd: Connected to MPD
```

**Comment by tardypad** [comment]
No only the one I mentioned before.

Usually those logs were ending with those 2 connection logs and the assert failure.
But this time, it just stopped there in the middle of a log
```
[2020-06-01 10:50:28.735]
```
with waybar still taking 100% CPU. I had to reload it then.

**Comment by jezcope** [comment]
I think I have the same problem as well: waybar crashes and dumps core after coming back from idle, and there are errors relating to mpd in there. I've just tried removing the mpd module, so if that fixes it then I'll add my stacktrace here.

**Comment by jezcope** [comment]
Ok, I'm running waybar with a systemd user unit, so journalctl captured the logs and the stack trace. I've edited it down from 125MB and put it here: [waybar.log](https://gist.github.com/jezcope/344c72f92eda49a0a674fccb4b07a4a4).

**Comment by jbenden** [comment]
Would someone please test the master branch, with #887 included?

Thanks,
-Joe

**Comment by Alexays** [comment]
Closed as #887 merged and likely fixed the pb.
Reopen the issue if it's not the case.
